functions/system.bash

```diff
@@ -264,71 +267,75 @@ srv_bind_mounts() {
+    if ! cond_redirect chmod --recursive go-rwx "$openhabHome"/.ssh; then echo "FAILED (set .ssh access)"; return 1; fi
   fi
 
-  if ! cond_redirect chown -R "${username:-openhabian}:${username:-openhabian}" "/home/${username:-openhabian}"; then echo "FAILED (${username:-openhabian} own $HOME)"; return 1; fi
+  if ! cond_redirect chown --recursive "${username:-openhabian}:${username:-openhabian}" "/home/${username:-openhabian}"; then echo "FAILED (${username:-openhabian} own $HOME)"; return 1; fi
 
   if ! cond_redirect setfacl -R --remove-all "${openhabFolders[@]}"; then echo "FAILED (reset file access)"; return 1; fi
   if ! cond_redirect setfacl -R -m g::rwX "${openhabFolders[@]}"; then echo "FAILED (set file access)"; return 1; fi
-  if cond_redirect setfacl -R -m d:g::rwX "${openhabFolders[@]}"; then echo "OK"; else echo "FAILED"; return 1; fi
+  if ! cond_redirect setfacl -R -m d:g::rwX "${openhabFolders[@]}"; then echo "FAILED"; return 1; fi
 
-  if ! cond_redirect chgrp root /var/log/samba /var/log/unattended-upgrades; then echo "FAILED (3rd party logdir)"; return 1; fi
+  if cond_redirect chgrp root /var/log/samba /var/log/unattended-upgrades; then echo "OK"; else echo "FAILED (3rd party logdir)"; return 1; fi
 
   if [[ -d /etc/homegear ]]; then
-    chown -R root:root /etc/homegear
-    find /etc/homegear -type d -print0 | xargs -0 chmod 755
-    find /etc/homegear -type f -print0 | xargs -0 chmod 644
-    find /etc/homegear -name "*.key" -print0 | xargs -0 chmod 644
-    find /etc/homegear -name "*.key" -print0 | xargs -0 chown homegear:homegear
-    chown homegear:homegear /etc/homegear/rpcclients.conf
-    chmod 400 /etc/homegear/rpcclients.conf
-    chown homegear:homegear -R /var/log/homegear
+    echo -n "$(timestamp) [openHABian] Applying additional file permissions recommendations for Homegear... "
+    if ! cond_redirect chown --recursive root:root /etc/homegear; then echo "FAILED (chown)"; return 1; fi
+    if ! (find /etc/homegear -type d -print0 | xargs -0 chmod 755); then echo "FAILED (chmod directories)"; return 1; fi
+    if ! (find /etc/homegear -type f -print0 | xargs -0 chmod 644); then echo "FAILED (chmod files)"; return 1; fi
+    if ! (find /etc/homegear -name "*.key" -print0 | xargs -0 chmod 644); then echo "FAILED (chmod *.key)"; return 1; fi
+    if ! (find /etc/homegear -name "*.key" -print0 | xargs -0 chown homegear:homegear); then echo "FAILED (chown *.key)"; return 1; fi
+    if ! cond_redirect chown homegear:homegear /etc/homegear/rpcclients.conf; then echo "FAILED (chown rpcclients)"; return 1; fi
+    if ! cond_redirect chmod 400 /etc/homegear/rpcclients.conf; then echo "FAILED (chmod rpcclients)"; return 1; fi
+    if ! cond_redirect chown --recursive homegear:homegear /var/log/homegear; then echo "FAILED (chown logs)"; return 1; fi
 
     # homeMatic/homegear controller HM-MOD-RPI-PCB uses GPIO 18 to reset HW
-    if ! [[ -d "${gpioDir}/gpio18" ]]; then
+    if ! [[ -d ${gpioDir}/gpio18 ]]; then
```

Comment:
huh? does shellcheck like that ?

Reply:
It doesn't care. Otherwise I would have a lot of ShellCheck error notifications on GitHub.